aosp_diff/linux_trusty/system/core/0025-Remove-or-modify-keystore-APIs-for-Clear-Linux.patch

```diff
@@ -0,0 +1,79 @@
+From 48aec5a0efbc38ed5f4607868317a4146d7151d3 Mon Sep 17 00:00:00 2001
+From: Chen Gang G <gang.g.chen@intel.com>
+Date: Thu, 11 Apr 2019 16:07:00 +0800
+Subject: [PATCH] Remove or modify keystore APIs for Clear Linux
+
+Some API is not available on clear linux
+
+Signed-off-by: Chen Gang G <gang.g.chen@intel.com>
+---
+ trusty/keymaster/legacy/trusty_keymaster_device.cpp | 20 ++++++++++++++++++++
+ 1 file changed, 20 insertions(+)
+
+diff --git a/trusty/keymaster/legacy/trusty_keymaster_device.cpp b/trusty/keymaster/legacy/trusty_keymaster_device.cpp
+index 13db10f..6aa51f5 100644
+--- a/trusty/keymaster/legacy/trusty_keymaster_device.cpp
++++ b/trusty/keymaster/legacy/trusty_keymaster_device.cpp
+@@ -146,11 +146,19 @@ keymaster_error_t TrustyKeymasterDevice::configure(const keymaster_key_param_set
+ 
+     AuthorizationSet params_copy(*params);
+     ConfigureRequest request(message_version_);
++
++   /* Get values of os_version and os_patchlevel from default implementation, then
++    * set them which are 0 to os_version and os_patchlevel
++   */
++#if 0
```

Comment:
"set the version values to 0 to make it compatible with clear linux"

Reply:
updated